MdeModulePkg/Library/VarCheckHiiLib/VarCheckHiiLib.inf

```diff
@@ -15,16 +15,18 @@
   MODULE_TYPE                    = DXE_RUNTIME_DRIVER
   VERSION_STRING                 = 1.0
   LIBRARY_CLASS                  = NULL|DXE_RUNTIME_DRIVER DXE_SMM_DRIVER
-  CONSTRUCTOR                    = VarCheckHiiLibNullClassConstructor
+  CONSTRUCTOR                    = VarCheckHiiLibSmmConstructor

```

Comment:
The lib also supports DXE_RUNTIME_DRIVER type. Why changing to VarCheckHiiLib**Smm**?